Dockerfile

```diff
@@ -18,7 +18,7 @@ RUN yum install -y epel-release &&\
 
 COPY ./requirements.txt /coreapi/
 RUN pushd /coreapi && \
-    pip3 install -r requirements.txt && \
+    python3.6 -m pip install -r requirements.txt && \
```

Comment:
@deepak1725 any particular reason for this change?

Reply:
@samuzzal-choudhury Build is failing. I mean Unit tests are failing without it. 
Please notice that with previous commit.